firestore/client.go

```diff
@@ -286,7 +292,10 @@ func (c *Client) Batch() *WriteBatch {
 }
 
 // commit calls the Commit RPC outside of a transaction.
-func (c *Client) commit(ctx context.Context, ws []*pb.Write) ([]*WriteResult, error) {
+func (c *Client) commit(ctx context.Context, ws []*pb.Write) (_ []*WriteResult, err error) {
+	ctx = trace.StartSpan(ctx, "cloud.google.com/go/firestore.Client.Commit")
```

Comment:
do we have a public Commit call or just this one? Commit is capitalized currently in the span name. Should this be `firestore.Client.commit`?

Reply:
We have a [public Commit](https://github.com/googleapis/google-cloud-go/blob/5344d93872074f8d66a0c8d0458f7bcf8bfae7fd/firestore/writebatch.go#L74) for WriteBatch. It contains this private `commit` method. 

But my logic then was that this `commit` method is also contained in the [`commit1`](https://github.com/googleapis/google-cloud-go/blob/5344d93872074f8d66a0c8d0458f7bcf8bfae7fd/firestore/client.go#L312) method. Which is used in 4 other public functions ([Create](https://github.com/googleapis/google-cloud-go/blob/5344d93872074f8d66a0c8d0458f7bcf8bfae7fd/firestore/docref.go#L130), [Set](https://github.com/googleapis/google-cloud-go/blob/5344d93872074f8d66a0c8d0458f7bcf8bfae7fd/firestore/docref.go#L162), [Delete](https://github.com/googleapis/google-cloud-go/blob/5344d93872074f8d66a0c8d0458f7bcf8bfae7fd/firestore/docref.go#L242), [Update](https://github.com/googleapis/google-cloud-go/blob/5344d93872074f8d66a0c8d0458f7bcf8bfae7fd/firestore/docref.go#L667)). But for these functions we don't have a span for this `Commit` rpc call. That's why I added it here.

I used capitalized `Commit` name because of the capitalized Commit RPC call name. If you think it should have lowercase name, then I'll update.

Reply:
I think I would prefer it to match the method name just for transparency. 👍

Reply:
Done